### PR TITLE
[Mobile] Budget month notes and budget action menu

### PR DESCRIPTION
**Pull request by joel-jeremy**
Make month notes and these action menus available in mobile:
image

### CODE REVIEW

**Comment by youngcw** [comment]
I thought this would be in the modal under the logo.  I feel like there isn't a big reason to have both the month name modal and the app logo modal.

**Comment by joel-jeremy** [comment]
Thanks for the feedback! Those are very good points. I pushed a commit to merge them into a single modal, please let me know if this is better than the previous iteration. Thanks!

**Comment by youngcw** [comment]
I do like this better.  Ill think some more on if I think is ok with just one menu.  Im really only wondering about the toggle hidden and new groups options.

I do like having the logo on the header, but it probably would make more sense to have month name be the button.  Maybe others can chime in

**Comment by joel-jeremy** [comment]
Maybe we can have the logo open the non-month specific menus (add new group, toggle hidden, etc) and the month open the month-specific ones (notes, actions, etc). We won't be needing the + on the header in this case.

**Comment by youngcw** [comment]
That could work

**Comment by joel-jeremy** [comment]
I have pushed an update to put the non-month related menus to the logo, and the month-related ones on the month header

**Comment by youngcw** [comment]
Nice.  I don't have any more complaints

File: packages/desktop-client/e2e/schedules.test.js-snapshots/Schedules-creates-a-new-schedule-posts-the-transaction-and-later-completes-it-1-chromium-linux.png
Comment (youngcw) [inline packages/desktop-client/e2e/schedules.test.js-snapshots/Schedules-creates-a-new-schedule-posts-the-transaction-and-later-completes-it-1-chromium-linux.png:1]: Im not sure why this one got updated, but I think that change matches what should be there now anyway

**Review by youngcw — APPROVED** [review]

File: packages/desktop-client/e2e/schedules.test.js-snapshots/Schedules-creates-a-new-schedule-posts-the-transaction-and-later-completes-it-1-chromium-linux.png
Comment (joel-jeremy) [reply packages/desktop-client/e2e/schedules.test.js-snapshots/Schedules-creates-a-new-schedule-posts-the-transaction-and-later-completes-it-1-chromium-linux.png:1]: That's odd. That always seem to appear on every VRT updates.

**Pull request merged**